backend/src/main/java/com/google/rolecall/jsonobjects/CastMemberInfo.java

```diff
@@ -27,14 +28,42 @@
 
   @Nullable
   @JsonIgnore
+  @JsonProperty("delete")
   public abstract Boolean delete();
 
-  @JsonCreator
-  public static CastMemberInfo create(@Nullable @JsonProperty("id") Integer id, 
-      @Nullable @JsonProperty("userId") Integer userId,
-      @Nullable @JsonProperty("subCastId") Integer subCastId,
-      @Nullable @JsonProperty("order") Integer order,
-      @Nullable @JsonProperty("delete") Boolean delete) {
-    return new AutoValue_CastMemberInfo(id, userId, subCastId, order, delete);
```

Comment:
We can have a create method call the builder and pass all the values to keep it backward compatible, but I am fine if we make these changes.

Reply:
I agree. I spoke to Jared about this just before he left and he told me he was just experimenting, but in retrospect he wished he had used the builder patter here too.

Reply:
Sounds good. Thanks for letting me know.